docs/index.html

```diff
@@ -312,22 +348,43 @@
       {
         quadrant: 1,
         ring: 0,
-        label: "Firebase",
+        label: "Ansible",
         active: false,
         moved: 0
       },
       {
         quadrant: 1,
         ring: 0,
-        label: "Grafana",
+        label: "Terraform",
         active: false,
         moved: 0
       },
       {
         quadrant: 1,
         ring: 0,
+        label: "Firebase",
+        active: false,
+        moved: 0
+      },
+      {
+        quadrant: 1,
+        ring: 3,
```

Comment:
Grafana put on `hold` is this intended?

Reply:
Thought we got the input from ops at the beginning of the review process, but I do not find where the input was given. Might be moved by mistake then

Reply:
reverted the change to Grafana